Anthony_Gabby_index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Code Academy Students</title>
+</head>
+
+<body>
+    <h1>Power Code Academy Students</h1>
+
+    <div class="developer">
+        <h2>Anthony Phillips</h2>
+            <img src="img/AnthonyPhillips.jpg"
+            width="100px"
+            height="100px">
+    </div>
+
+    <div class="developer">
+        <h2>Gabriella Massaro</h2>
+            <img src="img/GabriellaMassaro.jpg"
+            width="100px"
+            height="100px">
+    </div>
+
+    <div class="developer">
+        <h2>Liam Simmons</h2>
+             <img src="img/LiamSimmons.jpg"
+            width="100px"
+            height="100px">
+    </div>
+
+    <div class="developer">
+        <h2>Cayla Horsey</h2>
+       
+            <img src="img/CaylaHorsey.jpg"
+            width="100px"
+            height="100px">
+    </div>
```

Comment:
Would love to see some more consistent spacing/formatting. This would make it much easier to read and update. 
```suggestion
    <div class="developer">
        <h2>Cayla Horsey</h2>
        <img src="img/CaylaHorsey.jpg" width="100px" height="100px">
    </div>
```
This is just an example, but this applies throughout.

Reply:
Made the changes to address style